### DISCUSSION TITLE
View kanboards from different sites in one "view" or another kanboard

### DISCUSSION THREAD

**Discussion by astrakid** [kanboard/kanboard#5437] [Q&A]
hi,
i am using kanboards in different sites: for a voluntary position, for my job, for my private tasks.
is there a way to combine them all into a single view? e.g. i can connect with each login for a site and have it all in one view. i was not able ti find something yet, but maybe my search was bad.
regards,
andre

**Comment by fguillot** [comment]
I'm afraid that Kanboard hasn't been designed to provide such feature.

**Reply by astrakid** [reply]
ok. thanks.
kanboard is nevertheless a great tool.
